src/scripts/main.js

```diff
@@ -1 +1,41 @@
 'use strict';
+
+const addText = (number, isError) => `
+  ${number} promise was ${isError ? 'rejected' : 'resolved'}
+`;
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('mousedown', (e) => {
+    e.button === 0 && resolve(addText('First'));
+  });
+  setTimeout(() => reject(addText('First', 1)), 3000);
```

Comment:
In this function, you have a Boolean type as the second parameter `isError`, so pass that type there `true`/`false`